analysis/trigger_performance/country_template/docs/plotAtvCI.R

```diff
@@ -0,0 +1,102 @@
+  ## Script to plot the confidence intervals of probability of activation for individual triggers and activation timepoints
```

Comment:
It's true, I mean I think the best solution, if we are moving to separate repos for each country, is to have a separate repo for the report template, and a particular trigger report can be pinned to a specific version. This is the whole idea behind aa toolbox. Not sure how to do that with R exactly though. 

Because the problem right now in the AA repository (I imagine) is if you change any of the base files / functions that older reports link to, it could break them.